rollup.config.js

```diff
@@ -0,0 +1,72 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import replace from 'rollup-plugin-replace';
+import resolve from 'rollup-plugin-node-resolve';
+import uglify from 'rollup-plugin-uglify';
+
+import pkg from './package.json';
+
+function distBuild(options = {}) {
+    return {
+        input: 'src/index.js',
+        output: {
+            file: `dist/${options.filename}`,
+            format: options.format,
+            name: 'aphrodite',
+            sourcemap: options.sourcemap,
+        },
+        plugins: [
+            babel({
+                exclude: ['node_modules/**'],
+            }),
+            replace({
+                'process.env.NODE_ENV': JSON.stringify('production'),
+            }),
+            resolve({
+                browser: true,
+            }), // so rollup can find node modules
+            commonjs(), // so rollup can convert node modules to ESM if needed
+            options.minify && uglify(),
+        ]
+    };
+}
+
+function standardBuilds(filename) {
+    return {
+        input: `src/${filename}.js`,
+        external: [
+            ...Object.keys(pkg.dependencies),
+            'inline-style-prefixer/static/createPrefixer',
+            'inline-style-prefixer/static/plugins/calc',
+            'inline-style-prefixer/static/plugins/crossFade',
+            'inline-style-prefixer/static/plugins/cursor',
+            'inline-style-prefixer/static/plugins/filter',
+            'inline-style-prefixer/static/plugins/flex',
+            'inline-style-prefixer/static/plugins/flexboxIE',
+            'inline-style-prefixer/static/plugins/flexboxOld',
+            'inline-style-prefixer/static/plugins/gradient',
+            'inline-style-prefixer/static/plugins/imageSet',
+            'inline-style-prefixer/static/plugins/position',
+            'inline-style-prefixer/static/plugins/sizing',
+            'inline-style-prefixer/static/plugins/transition',
+        ],
+        output: [
+            { file: `lib/${filename}.js`, format: 'cjs' },
+            { file: `lib/${filename}.mjs`, format: 'es' },
+        ],
+        plugins: [
+            babel({
+                exclude: ['node_modules/**'],
+            }),
+            commonjs(), // so rollup can convert node modules to ESM if needed
+        ]
+    };
+}
+
+export default [
+    distBuild({ filename: 'aphrodite.umd.js', format: 'umd', sourcemap: true, minify: false }),
+    distBuild({ filename: 'aphrodite.umd.min.js', format: 'umd', sourcemap: true, minify: true }),
+    distBuild({ filename: 'aphrodite.js', format: 'cjs', sourcemap: false, minify: false }),
+    standardBuilds('index'),
```

Comment:
just to check my understanding, is this going to build bundles into `lib/` or leave things in individual files like they are currently?

Reply:
This will build bundles into `lib/` without the individual files. Here's what my `lib/` directory looks like after building like this:

```
total 504
drwxr-xr-x  12 joe_lencioni  staff   384B Oct 12 11:30 .
drwxr-xr-x  31 joe_lencioni  staff   992B Oct 13 10:54 ..
-rw-r--r--   1 joe_lencioni  staff   5.8K Oct 12 10:46 exports.js
-rw-r--r--   1 joe_lencioni  staff    14K Oct 12 10:46 generate.js
-rw-r--r--   1 joe_lencioni  staff    46K Oct 12 11:30 index.js
-rw-r--r--   1 joe_lencioni  staff    45K Oct 12 11:30 index.mjs
-rw-r--r--   1 joe_lencioni  staff    11K Oct 12 10:46 inject.js
-rw-r--r--   1 joe_lencioni  staff    45K Oct 12 11:30 no-important.js
-rw-r--r--   1 joe_lencioni  staff    44K Oct 12 11:30 no-important.mjs
-rw-r--r--   1 joe_lencioni  staff   4.4K Oct 12 10:46 ordered-elements.js
-rw-r--r--   1 joe_lencioni  staff   4.4K Oct 12 10:46 staticPrefixData.js
-rw-r--r--   1 joe_lencioni  staff   6.0K Oct 12 10:46 util.js
```

Reply:
I've updated this PR to build the cjs version into lib/ and the esm version into es/